go/glacier/README.md

```diff
@@ -0,0 +1,74 @@
+# AWS SDK for Go code examples for Amazon Simple Storage Service Glacier
```

Comment:
... for **Amazon S3 Glacier**
(use short name in the title for brevity)

Reply:
Fixed